.github/workflows/waspc-ci.yaml

```diff
@@ -47,7 +47,7 @@ jobs:
           # We can return to `ubuntu-latest` when enough time has elapsed.
           # Still looking into musl static binaries.
           # Ref: https://github.com/wasp-lang/wasp/issues/650
-          - ubuntu-20.04
+          - ubuntu-latest
```

Comment:
@sodic there is a big fat comment above that you should take into account and/or update/delete!

Reply:
I've set the ubuntu version to ubuntu-22.04.

How do I check this doesn't have the glibc problem and whether it's even necessary to use the older version of ubuntu?

Reply:
> You have set it to ubuntu-latest though, not 22.04, at least excplicitly, right?

Hm, I think you're looking at old code.

I understand the problem, but am wondering how to test it:
> How do I check this doesn't have the glibc problem and whether it's even necessary to use the older version of ubuntu?